tsconfig.common.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "lib": ["ES2019", "dom"],
+    "lib": ["ES2019", "dom", "dom.iterable"],
```

Comment:
Needed in order to register URLSearchParams as an iterable object for `buildPathWithGlobalParams` in `navigation.ts`.